docs/COMPATIBILITY.md

```diff
@@ -326,6 +326,12 @@ Note that MS SQL Server and Oracle are not supported as database backends.
 		<td>TBD</td>
 		<td></td>
 	</tr>
+	<tr>
+		<th><a href="https://www.semantic-mediawiki.org/wiki/Semantic_MediaWiki_4.1.3">4.1.3</a></th>
+		<td>7.10.x</td>
+		<td>Not supported (TBD)</td>
+		<td><a href="https://github.com/SemanticMediaWiki/SemanticMediaWiki/issues/5611#issuecomment-1962729300">Confirmed to work with ElasticSearch 7.10.2 (MW 1.39)</a></td>
+	</tr>
 	<tr>
 		<th>4.1.x</th>
```

Comment:
I'd change this to 4.1.0 - 4.1.2

Reply:
I did not understand where you are referring to. Is it in the current line of the table? 

![2024-03-01_16-35](https://github.com/SemanticMediaWiki/SemanticMediaWiki/assets/3636594/a558e7f7-eb8d-47f9-9089-c95824d661ea)

Reply:
The review interface deteriorated. I was unable to comment on your change somehow. Since you added 4.1.3, 4.1.x could be changed as suggested.

Reply:
Done